Project.toml

```diff
@@ -2,3 +2,6 @@ name = "TransitionIndicators"
 uuid = "5f5b98ec-1183-43e0-887a-12fdc55c52f7"
 authors = ["Jan Swierczek-Jereczek <jan.jereczek@gmail.com>", "George Datseris <datseris.george@gmail.com>"]
 version = "0.1.0"
+
+[deps]
+Revise = "295af30f-e4ad-537b-8983-00126c2a3abe"
```

Comment:
Revise has no reason to be a dependency. Load revise in your global project before doing anything else.